pkg/util/allowed_users.go

```diff
@@ -0,0 +1,45 @@
+package util
+
+type AllowedUsers struct {
```

Comment:
How do you feel about renaming this struct (including variables and file names) to `AllowedTenants`? - I think at this point we've settled on the word tenant in preference over users.

Reply:
SGTM, good idea.